enterprise/utils.py

```diff
@@ -1426,85 +1440,77 @@ def is_user_enrolled(user, course_id, course_mode, enrollment_client=None):
     return False
 
 
-def enroll_user(enterprise_customer, user, course_mode, *course_ids, **kwargs):
```

Comment:
this function is really only used to enroll in one course, and the succeeded flag also becomes clearer in this single course mode hence I changed it